bin/fuel-core/src/cli/run.rs

```diff
@@ -714,6 +714,7 @@ impl Command {
                 max_tx_update_subscriptions: tx_number_active_subscriptions,
                 subscription_ttl,
                 status_cache_ttl: status_cache_ttl.into(),
+                metrics: disabled_metrics.is_enabled(Module::TxStatusManager),
```

Comment:
(can be addressed in followup)

`disabled_metrics.is_enabled()`

doesn't make too much sense right now, maybe the function should be `.includes()` to be clearer :)

Reply:
Yup, this part ended up being somewhat goofy :)
I think `!disabled_metrics.includes()` while a bit more idiomatic may be actually more difficult to parse for humans.

I considered adding a helper function `is_metric_enabled()` but it didn't improve much in my opinion, so I'll leave this as is for now to not increase confusion for reviewers.